src/style.css

```
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  max-width: 100%;
+  padding: 0 50px;
+  --accent-color: #00acdc;
+}
+
+.logo {
+  padding-top: 5px;
+}
+
+.header__list {
+  display: flex;
+}
+
+.nav__item {
+  list-style: none;
+  display: flex;
+  padding-left: 20px;
```

Should be margin-left instead, because it's not an inner area of an element. It sets a distance between two elements
```suggestion
  margin-left: 20px;
```